frontend/javascripts/oxalis/view/right-border-tabs/advanced_search_popover.tsx

```diff
@@ -7,6 +7,8 @@ import Shortcut from "libs/shortcut_component";
 import DomVisibilityObserver from "oxalis/view/components/dom_visibility_observer";
 import { mod } from "libs/utils";
 
+const PRIMARY_COLOR = "var(--ant-color-primary)";
```

Comment:
Uh nice :tada:, this made me come up with the following possible refactoring: https://github.com/scalableminds/webknossos/issues/8150

Reply:
I LOVE THE IDEA, THANK YOU :raised_hands: